lesson04/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+document.addEventListener('DOMContentLoaded', function() {
+    const attributes = [
+        {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+        {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+    ]
+
+    const ul = document.getElementById('js-lists');
+    const arrayLength = attributes.length;
+
+    const fragment = document.createDocumentFragment();
+
+    let i = 0;
+    while (i < arrayLength) {
+
+        const attribute = attributes[i];
+        const li = document.createElement('li');
+        const anchor = document.createElement('a');
+        const img = document.createElement('img');
+
+        anchor.textContent = attribute.text;
+        anchor.href = '/' + attribute.to;
```

Comment:
Nit
you can also write like blow.

```suggestion
        anchor.href = `/${attribute.to}`;
```

Reply:
Thank you for showing me a new way to write.
 I used it right away!